packages/inventory/src/api/api.js

```diff
@@ -78,6 +78,35 @@ export function getEntities(items, {
 }, showTags) {
     if (hasItems && items.length > 0) {
         return hosts.apiHostGetHostById(items, undefined, perPage, page, undefined, undefined, { cancelToken: controller && controller.token })
+        .then(async (data) => {
```

Comment:
No need to use `then`if we are using `async/wait`
```JS
const invData = await hosts.apiHostGetHostById(items, undefined, perPage, page, undefined, undefined, { cancelToken: controller && controller.token });
return {
   ...invData,
   ...Object.keys(fields).length > 0 && {
    results: mergeArraysByKey([
     invData.results,
     (await hosts.apiHostGetHostSystemProfileById(
       items,
       perPage,
       page,
       orderBy,
       undefined,
       undefined,
       undefined,
       {
         cancelToken: controller && controller.token,
         query: { ...generateFields(fields) }
       }
     })).results
    ])
   }
}
```

Reply:
Ok, I didn't want to make so big refactor here 😅